elevenlabs_mcp/server.py

```
@@ -0,0 +1,900 @@
+
+    Voice preview files are saved as: voice_design_(generated_voice_id)_(timestamp).mp3
+
+    Example file name: voice_design_Ya2J5uIa5Pq14DNPsbC1_20250403_164949.mp3
+
+    ⚠️ COST WARNING: This tool makes an API call to ElevenLabs which may incur costs. Only use when explicitly requested by the user.
+    """
+)
+@_safe_api
+def text_to_voice(
+    voice_description: str,
+    text: str = None,
```

Ruff (0.15.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)